.licensed.yaml

```diff
@@ -0,0 +1,20 @@
+sources:
```

Comment:
Do we need to add this file if `.drone.yaml` doesn't have the license-check step?
Or perhaps you were looking to add it in later?

Reply:
my bad, let me add the step in the pipeline

Reply:
done